### PR TITLE
Python 2/3 compatible code

### PR DESCRIPTION
**Pull request by snowman2**
This needs to be tested, but it is a good start.

Note: This is only the source code for tethys and not the dependencies.

See:
https://github.com/tethysplatform/tethys/issues/262

### CODE REVIEW

**Comment by snowman2** [comment]
Issue #242 needs to be fixed before the merge can happen.

**Comment by snowman2** [comment]
Installation worked:
```
$ bash install_tethys.sh --python-version 3
```
NOTE: Installation will only work if you merge the code. Otherwise, tweaking needed.

File: environment_py3.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- pyopenssl
+- pip:
+    - django-gravatar2
+    - django-bootstrap3
+    - django-model-utils
+    - django-guardian
+    - django-simple-captcha
+    - django-termsandconditions
+    - tethys_dataset_services
+    # - condorpy
```
Comment (sdc50) [inline environment_py3.yml]: I'm just about to release condorpy so lets add this back in

File: scripts/install_tethys.sh [hunk 2]
```diff
@@ -219,10 +225,20 @@ then
 
     # create conda env and install Tethys
     echo "Setting up the ${CONDA_ENV_NAME} environment..."
-    conda env create -n ${CONDA_ENV_NAME} -f environment_py2.yml
+    conda env create -n ${CONDA_ENV_NAME} -f "environment_py${PYTHON_VERSION}.yml"
     . activate ${CONDA_ENV_NAME}
     python setup.py develop
 
+    if [ $PYTHON_VERSION == '3' ]
```
Comment (sdc50) [inline scripts/install_tethys.sh]: with condorpy released we can remove this section

**Pull request merged**